### PR TITLE
Feat: Add spanish language translation

### PR DESCRIPTION
**Pull request by letelete**
Fixes #Issue_Number

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by luckyman20** [comment]
@letelete Please upload few screenshots too.

**Comment by letelete** [comment]
Sure, will do.

**Comment by letelete** [comment]
image

image

image

**Review by luckyman20 — APPROVED** [review]
Looks good to me, let the Travis build and I will approve your task too.

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -1,272 +1,420 @@
...
+    <string name="about_us">Sobre nosotros</string>
+    <string name="saving_account_details">Guardar detalles de la cuenta</string>
+    <string name="savings_account">Cuenta de ahorros</string>
+    <string name="error_loan_account_details_loading">Error al cargar la información de la cuenta de crédito</string>
+    <string name="nominal_interest_rate">Tasas de interés nominales</string>
+    <string name="account_number">Numero de cuenta</string>
+    <string name="account_balance">Saldo de cuenta</string>
+    <string name="total_deposits">Depósitos totales</string>
+    <string name="open_drawer">Abre el panel de navegación</string>
+    <string name="close_drawer">Cerrar el panel de recuperación</string>
```
Comment (migueldinisdev) [inline app/src/main/res/values-es/strings.xml]: Why did you use different words for the same translation? "navegación" and "recuperación"?

Comment (letelete) [reply app/src/main/res/values-es/strings.xml]: Looks like a silly mistake, thanks for pointing it out!

**Comment by miPlodder** [comment]
@letelete Good job 👍

**Pull request merged**